setup.py

```diff
@@ -30,12 +35,15 @@
         subprocess.call(["git", "submodule", "update", "--init", "--recursive"])
     if not os.path.exists("vcpkg/buildtrees"):
         subprocess.call(["git", "pull"], cwd="vcpkg")
+        args = ["install"]
+        if VCPKG_TRIPLET is not None:
+            args.append( f"--triplet={VCPKG_TRIPLET}" )
```

Comment:
no excess spaces between paren

Reply:
we should update the linter for these cases

Reply:
`ruff` doesnt raise an error like `black` would, but `make fix` should adjust it